zapier/test/creates.test.js

```diff
@@ -0,0 +1,55 @@
+/* globals describe, expect, test */
+
+const zapier = require('zapier-platform-core');
+
+const App = require('../index');
+const appTester = zapier.createAppTester(App);
+zapier.tools.env.inject();
+
+
+const FILE_URL =
+  'https://cdn.zapier.com/storage/files/f6679cf77afeaf6b8426de8d7b9642fc.pdf';
```

Comment:
Is this file supposed to be hardcoded into the code?

Reply:
Yes - it's only used for testing